app/src/main/res/layout/activity_account.xml

```diff
@@ -10,77 +10,93 @@
 
     <android.support.design.widget.AppBarLayout
         android:layout_width="match_parent"
+        android:theme="@style/AppTheme"
```

Comment:
I had to enforce the dark theme for this area, because black icons don't work on image backgrounds